playbooks/roles/airship-setup-deployer/tasks/cfssl-tls-certs.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Creates CloudFare TLS certs scripts directory
+  file:
+    path: "{{ suse_tls_certs_dir }}/scripts"
+    state: directory
+
+- name: Check for existing TLS cert files presence
+  stat:
+    path: "{{ item }}"
+  register: _cfssl_certs_stat
+  with_items:
+    - "{{ suse_tls_certs_dir }}/scripts/ca.pem"
+    - "{{ suse_tls_certs_dir }}/scripts/ca-key.pem"
+    - "{{ suse_tls_certs_dir }}/scripts/server.pem"
+    - "{{ suse_tls_certs_dir }}/scripts/server-key.pem"
+
+- name: Copy CSR and config files to CloudFare scripts directory
+  template:
+    src: "{{ item }}"
+    dest: "{{ suse_tls_certs_dir }}/scripts/{{ (item | splitext)[0] }}"
+  with_items:
+    - tls-ca-csr.json.j2
+    - tls-ca-config.json.j2
+    - tls-server-csr.json.j2
+  register: _cfssl_config
+
+- name: Check if need to generate certs
+  set_fact:
+    _generate_cfssl_certs: True
+  when: ( ( _cfssl_certs_stat.results | selectattr('stat.exists') | map(attribute='stat')| list | length !=4)
+          or (_cfssl_config.changed ) )
+
+- name: Download latest stable cfssl (CloudFare pki/tls toolkit)
+  become: yes
+  shell: >-
+    ./cfssl || (curl -L https://pkg.cfssl.org/R1.2/cfssl_linux-amd64 -o cfssl &&
```

Comment:
we are moving things to rpm package based. I could only find this: https://build.opensuse.org/package/show/home:darix:playground/cfssl

Reply:
As you mentioned as well, I cannot locate a published/supported opensuse rpm package for cfssl. we can look into building a new package in OBS and adding usage thereafter.